### ISSUE TITLE
New Tribler GUI crashes when adding torrent from URL

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#2661]
##### Tribler version/branch+revision: #####

v7.0.0-alpha1 6bda08fab632a38fbb3fc4e9f4073e23b2d8df7a

##### Operating system and version: #####

Linux

##### Steps to reproduce the behavior: #####

1. Start Tribler core with `twistd -n tribler`
2. Start GUI with `./tribler.sh`
3. Press "+" button.
4. "Import torrent from URL"
5. Paste http URL to torrent file
6. Press "add"
7. Window appears with "Loading files..."

##### Expected behavior: #####

It shows the list of files

##### Actual behavior: #####

GUI crashes.

##### Relevant log file output: #####

```
Traceback (most recent call last):
  File "/home/tribler/tribler/TriblerGUI/event_request_manager.py", line 85, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback: <type 'exceptions.TypeError'>: 'NoneType' object has no attribute '__getitem__'
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:588:_runCallbacks
/home/tribler/tribler/Tribler/Core/Modules/restapi/torrentinfo_endpoint.py:69:_on_loaded
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:393:callback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:501:_startRunCallbacks
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:588:_runCallbacks
/home/tribler/tribler/Tribler/Core/Modules/restapi/torrentinfo_endpoint.py:49:on_got_metainfo
]
```

@devos50

**Comment by vi** [comment]
Adding works better. But still can crash:

But still crashes:

```
Traceback (most recent call last):
  File "/home/tribler/tribler/TriblerGUI/event_request_manager.py", line 85, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/twisted/application/app.py", line 390, in startReactor
    self.config, oldstdout, oldstderr, self.profiler, reactor)
  File "/usr/lib/python2.7/dist-packages/twisted/application/app.py", line 311, in runReactorWithLogging
    reactor.run()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1194, in run
    self.mainLoop()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 798, in runUntilCurrent
    f(*a, **kw)
  File "/home/tribler/tribler/Tribler/community/multichain/community.py", line 410, in on_tunnel_remove
    if self._public_key > candidate.get_member().public_key:
exceptions.AttributeError: 'NoneType' object has no attribute 'public_key'
```

**Comment by synctext** [comment]
Thnx @vi!
Still crashes on Windows. This big new version takes a while to get stable..

**Comment by devos50** [comment]
First error should be fixed by #2649, I've seen the second error various times in crash reports already but I'm not sure whether we already have an issue for it. Can you create one if there isn't? Thanks!

**Comment by vi** [comment]
@devos50, Done: #2664
